### ISSUE TITLE
[Table] Remove deprecated attributes when pasting

### ISSUE THREAD

**Issue by renow-luxembourg** [adobe/aem-core-wcm-components#2906]
## Feature Request

In UE, for accessibility audit we use the RGAA referentiel, for the data tables attributes it's a little stricter than WCAG.

**Is your feature request related to a problem? Please describe.**
When a user copies/pastes a table element from Excel or Word, it comes with some deprecated and unwanted attributes (for example : align, cellpadding, cellspacing, etc.)
During accessibility audits, when encountering this type of content, the RGAA 10.1.2 test failed.
Test 10.1.2 translation: "On each web page, attributes used for information presentation must not be available in the generated source code. Has this rule been followed?"
The complete list of deprecated attributes can be found in the RGAA testing methodology

**Describe the solution you'd like**
The simplest solution will be to automatically remove these attributes when the user pastes the content.

**Are there alternatives?**
We can also develop a Rich Text Editor plugin that adds a button to the RTE toolbar. The action of this button will be to remove all the deprecated attributes.

**Documentation**
- HTML5 obsolete attributes https://www.w3.org/TR/2014/REC-html5-20141028/obsolete.html
- RGAA 10.1.2 : https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#10.1.2 (In french)
- RAWeb 10.1.2 : https://accessibilite.public.lu/fr/raweb1/criteres.html#test-10-1-2 (In french)
